src/assets/i18n/en.json5

```diff
@@ -6316,4 +6297,336 @@
+
+  "admin-notify-dashboard.metrics": "Metrics",
+
+  "admin-notify-dashboard.received-ldn": "Number of received LDN",
+
+  "admin-notify-dashboard.generated-ldn": "Number of generated LDN",
+
+  "admin-notify-dashboard.NOTIFY.incoming.accepted": "Accepted",
+
+  "admin-notify-dashboard.NOTIFY.incoming.accepted.description": "Accepted inbound notifications",
+
+  "admin-notify-logs.NOTIFY.incoming.accepted": "Currently displaying: Accepted notifications",
+
+  "admin-notify-dashboard.NOTIFY.incoming.processed": "Processed LDN",
+
+  "admin-notify-dashboard.NOTIFY.incoming.processed.description": "Processed inbound notifications",
+
+  "admin-notify-logs.NOTIFY.incoming.processed": "Currently displaying: Processed LDN",
+
+  "admin-notify-logs.NOTIFY.incoming.failure": "Currently displaying: Failed notifications",
+
+  "admin-notify-dashboard.NOTIFY.incoming.failure": "Failure",
+
+  "admin-notify-dashboard.NOTIFY.incoming.failure.description": "Failed inbound notifications",
+
+  "admin-notify-logs.NOTIFY.outgoing.failure": "Currently displaying: Failed notifications",
+
+  "admin-notify-dashboard.NOTIFY.outgoing.failure": "Failure",
+
+  "admin-notify-dashboard.NOTIFY.outgoing.failure.description": "Failed outbound notifications",
+
+  "admin-notify-logs.NOTIFY.incoming.untrusted": "Currently displaying: Untrusted notifications",
+
+  "admin-notify-dashboard.NOTIFY.incoming.untrusted": "Untrusted",
+
+  "admin-notify-dashboard.NOTIFY.incoming.untrusted.description": "Inbound notifications not trusted",
+
+  "admin-notify-logs.NOTIFY.incoming.delivered": "Currently displaying: Delivered notifications",
+
+  "admin-notify-dashboard.NOTIFY.incoming.delivered.description": "Inbound notifications successfully delivered",
+
+  "admin-notify-dashboard.NOTIFY.outgoing.delivered": "Delivered",
+
+  "admin-notify-logs.NOTIFY.outgoing.delivered": "Currently displaying: Delivered notifications",
+
+  "admin-notify-dashboard.NOTIFY.outgoing.delivered.description": "Outbound notifications successfully delivered",
+
+  "admin-notify-logs.NOTIFY.outgoing.queued": "Currently displaying: Queued notifications",
+
+  "admin-notify-dashboard.NOTIFY.outgoing.queued.description": "Notifications currently queued",
+
+  "admin-notify-dashboard.NOTIFY.outgoing.queued": "Queued",
+
+  "admin-notify-logs.NOTIFY.outgoing.queued_for_retry": "Currently displaying: Queued for retry notifications",
+
+  "admin-notify-dashboard.NOTIFY.outgoing.queued_for_retry": "Queued for retry",
+
+  "admin-notify-dashboard.NOTIFY.outgoing.queued_for_retry.description": "Notifications currently queued for retry",
+
+  "admin-notify-dashboard.NOTIFY.incoming.involvedItems": "Involved items",
+
+  "admin-notify-dashboard.NOTIFY.incoming.involvedItems.description": "Items related to inbound notifications",
+
+  "admin-notify-dashboard.NOTIFY.outgoing.involvedItems": "Involved items",
```

Comment:
Same here. I think it might be better to call this "Items Involved" so that it matches the search results header